### PR TITLE
String feedback_manual_toast_text and the reference to it has been updated

### LINKED ISSUE #10029: Misleading toast message on clicking "Send troubleshooting report"
A toast containing this message is displayed on clicking "Send troubleshooting report":
`AnkiDroid has encountered a problem; a report is being sent to the developers`

Can we use better terminology here? The above message makes it seem like that the app crashed or something went wrong even though the user themselves clicked the troubleshoot option to send a report. It is possible that AnkiDroid encountered no problem and the user wanted to report some other issue.

### PR DESCRIPTION
**Pull request by AayushSolanki**
Fixes #10029
String feedback_manual_toast_text has been edited with appropriate message and the reference to it is also updated.

### CODE REVIEW

File: AnkiDroid/src/main/res/values/05-feedback.xml [hunk 1]
```diff
@@ -27,7 +27,7 @@
     <string name="feedback_title">AnkiDroid Feedback</string>
     <string name="feedback_default_text">Enter your feedback or information about the nature of any errors you’re reporting.</string>
     <string name="feedback_auto_toast_text">AnkiDroid has encountered a problem; a report is being sent to the developers…</string>
-    <string name="feedback_manual_toast_text">AnkiDroid has encountered a problem; a report is being generated…</string>
+    <string name="feedback_for_manual_toast_text">An error report is being prepared for the developers…</string>
```
Comment (Akshay0701) [inline AnkiDroid/src/main/res/values/05-feedback.xml:30]: can I get to know why did you mention the `error report`, did we encounter any bug while showing this toast message? 
to what I understand by issues discussion is our main aim for this fix is to ensure users that there is no problem, right?

According to @ShridharGoel from [comment](https://github.com/ankidroid/Anki-Android/issues/10029#issuecomment-1005136940), I think we should use `A report is being generated for the reference of developers`

**Review by Akshay0701 — CHANGES_REQUESTED** [review]
Hey @AayushSolanki,
Thank you for your contribution...

I have one question on feedback_for_manual_toast_text

File: AnkiDroid/src/main/res/values/05-feedback.xml (hunk 1 above)
Comment (mikehardy) [reply AnkiDroid/src/main/res/values/05-feedback.xml:30]: @Akshay0701 this was by specific request from the first iteration of this PR if I understand correctly: https://github.com/ankidroid/Anki-Android/pull/10120#pullrequestreview-852502937

File: AnkiDroid/src/main/java/com/ichi2/anki/AnkiDroidApp.java [hunk 2]
```diff
@@ -597,7 +597,7 @@ public void setAcraReportingMode(String value) {
                 toastBuilder.setResText(R.string.feedback_auto_toast_text);
             } else if (value.equals(FEEDBACK_REPORT_ASK)) {
                 dialogBuilder.setEnabled(true);
-                toastBuilder.setResText(R.string.feedback_manual_toast_text);
+                toastBuilder.setResText(R.string.feedback_for_manual_toast_text);
```
Comment (Akshay0701) [inline AnkiDroid/src/main/java/com/ichi2/anki/AnkiDroidApp.java:600]: I am also curious to know the reason why you changed the `feedback_manual_toast_text` -> `feedback_for_manual_toast_text` 

previous names looks good to me.

Comment (mikehardy) [reply AnkiDroid/src/main/java/com/ichi2/anki/AnkiDroidApp.java:600]: we needed to force a re-translation, to do that you have to change the key

Comment (Akshay0701) [reply AnkiDroid/src/main/java/com/ichi2/anki/AnkiDroidApp.java:600]: Oh okay🤝

**Comment by Akshay0701** [comment]
Some issues:
```
> Task :AnkiDroid:compileAmazonDebugAndroidTestJavaWithJavac FAILED
/Users/runner/work/Anki-Android/Anki-Android/AnkiDroid/src/androidTest/java/com/ichi2/anki/tests/ACRATest.java:131: error: cannot find symbol
        assertToastMessage(R.string.feedback_manual_toast_text);
                                   ^
  symbol:   variable feedback_manual_toast_text
  location: class string
/Users/runner/work/Anki-Android/Anki-Android/AnkiDroid/src/androidTest/java/com/ichi2/anki/tests/ACRATest.java:214: error: cannot find symbol
        assertToastMessage(R.string.feedback_manual_toast_text);
                                   ^
  symbol:   variable feedback_manual_toast_text
  location: class string
/Users/runner/work/Anki-Android/Anki-Android/AnkiDroid/src/androidTest/java/com/ichi2/anki/tests/ACRATest.java:226: error: cannot find symbol
        assertToastMessage(R.string.feedback_manual_toast_text);
                                   ^
  symbol:   variable feedback_manual_toast_text
  location: class string
3 errors
```

**Review by Akshay0701 — APPROVED** [review]
@AayushSolanki Thanks,
LGTM, wanting for Shridhar's review...

**Review by ShridharGoel — APPROVED** [review]
Looks good.

**Review by ShridharGoel — APPROVED** [review]
Looks good.

**Comment by mikehardy** [comment]
Hi there! **JANUARY** OpenCollective Notice

Just a friendly notice that we try to process OpenCollective payments monthly - it's time for [January 2022 submissions](https://github.com/ankidroid/Anki-Android/pulls?q=is%3Apr+merged%3A2022-01-01..2022-01-31) (I'll follow this with the same process for February in a week)

If you are interested in compensation for this work, the process with details is here:

https://github.com/ankidroid/Anki-Android/wiki/OpenCollective-Payment-Process#how-to-get-paid

(I only post one comment per person to avoid spamming you, regardless of the number of PRs merged, but this note applies to all PRs merged for in the month of January)

Thanks!

**Comment by AayushSolanki** [comment]
> Hi there! **JANUARY** OpenCollective Notice
> 
> Just a friendly notice that we try to process OpenCollective payments monthly - it's time for [January 2022 submissions](https://github.com/ankidroid/Anki-Android/pulls?q=is%3Apr+merged%3A2022-01-01..2022-01-31) (I'll follow this with the same process for February in a week)
> 
> If you are interested in compensation for this work, the process with details is here:
> 
> https://github.com/ankidroid/Anki-Android/wiki/OpenCollective-Payment-Process#how-to-get-paid
> 
> (I only post one comment per person to avoid spamming you, regardless of the number of PRs merged, but this note applies to all PRs merged for in the month of January)
> 
> Thanks!

Thank you @mikehardy for the notice but i have contributed only for the sake of gaining knowledge and to give my contribution to the app and will be definitely contributing in future i don't want any compensation for my contribution.

**Pull request merged**